inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/region_yolo_gpu_ref.cl

```diff
@@ -12,94 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "include/common.cl"
-#include "include/data_types.cl"
+#include "include/include_all.cl"
 
+#if INPUT0_LAYOUT_BFYX
```

Comment:
What about using macro such as `INPUT0_SIZE_X`, `INPUT0_FEATURE_NUM`? Then, you don't need to define IW, IH, IC, IB additionally.